src/web-stack/apps/positive-affirmations/src/app/users/entities/affirmation.entity.ts

```diff
@@ -0,0 +1,35 @@
+import { Entity, PrimaryGeneratedColumn, Column } from 'typeorm';
+import { UserEntity } from './user.entity'
+
+@Entity({ name: 'user' })
+export class AffirmationEntity {
+    @PrimaryGeneratedColumn('uuid')
+    public id: string;
+
+    // Reference for column types to database compatibility:
+    // https://github.com/typeorm/typeorm/blob/master/src/driver/types/ColumnTypes.ts
+    @Column('text')
+    public title: string;
+
+    @Column('text', {nullable: true})
+    public note: string;
+
+    @Column('int')
+    public likes: number;
+
+    // TODO how to do this?
+    // @Column({name: 'created_by', type: 'user'})
+    // public createdBy: string;
```

Comment:
We could define the type of createdBy to be a user entity and do a many-to-one relationship on it. And over at the user entity definition, we can define a list of affirmations and do a one-to-many relationship there

You can check here for more on how we can make relationships work:
- https://docs.nestjs.com/techniques/database#relations
- https://stackoverflow.com/questions/60378494/typeorm-foreign-key
-